index.html

```diff
@@ -469,16 +272,69 @@ <h2 class="contact-heading-details color-white">Contact me</h2>
         </div>
       </footer>
     </div>
+    
+
+ <!-- popup session -->
+
+     <!-- popup -->
+    <div class="popupContainer">
```

Comment:
- [ ] The popup window should follow the Figma template design, including the fonts. colors and images. It would be nice to follow the design template.

Figma Design
![image](https://user-images.githubusercontent.com/81932478/168931716-702ee890-470f-4de0-8848-2910c9ce0f99.png)

Your Design
![image](https://user-images.githubusercontent.com/81932478/168931779-b755d540-e10a-4db8-ae7c-7aa628f0329f.png)